src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -120,12 +120,12 @@ public class C
           ""locations"": [
           ],
           ""fullMessage"": ""Program does not contain a static 'Main' method suitable for an entry point"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Error"",
```

Comment:
Here's an example. You should zap severity from the property dictionary and populate 'kind'.